kubernetes/README.md

```diff
@@ -4,70 +4,111 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
+## Deploy the SCIM bridge 
+
+Using the `op-scim-deployment.yaml` and `op-scim-service.yaml` files, deploy the 1Password SCIM bridge to your Kubernetes cluster.
+
+These files configure the 1Password SCIM Bridge to connect to the redis instance indicated by the args, and it deploys a load balancer to handle traffic on ports 80 and 443. Traffic on :80 is needed to perform the LetsEncrypt certificate challenges, after which all SCIM traffic will be served on :443.
+
 ```
-kubectl create secret generic scimsession --from-file=./scimsession
+kubectl apply -f op-scim-deployment.yaml
+kubectl apply -f op-scim-service.yaml
 ```
-Make sure to pass the filepath of the scimsession file that you downloaded.  The above command will look for the file in  this folder (the `/kubernetes/` folder) of the repository.
 
-## Deploy the SCIM bridge
 
-Using the `op-scim-deployment.yaml` and `op-scim-service.yaml` files, deploy the 1Password SCIM bridge to your Kubernetes cluster.
+## Finish configuring DNS
 
-NOTE: In order to obtain an SSL/TLS certificate for your SCIM Bridge instance, you must include a domain name in the `containers.args` field in the `op-scim-deployment.yaml` file. This will only succeed if a DNS record exists that points to your GCP load balancer. Please read the section about DNS records at the beginning of this example.
+As mentioned before, you'll need to configure you DNS after the fact, as configuring DNS requires the IP address of the deployed container.
 
-NOTE: If you are using an existing redis instance that is not running on `redis:6379`, add the `--redis-host=[host]` and `--redis-port=[port]` flags to `containers.args` in the deployment yaml file.
+It's important that you use the IP address of the container that runs the load balancer. The load balancer is what handles traffic for ports 80 and 443 for the SCIM bridge. Here are the mappings:
 
-Example:
 ```
-kubectl apply -f op-scim-deployment.yaml
-kubectl apply -f op-scim-service.yaml
+Load Balancer: 80 -> SCIM Bridge: 8080
+Load Balancer: 443 -> SCIM Bridge: 8443
 ```
 
-These files configure the 1Password SCIM Bridge to connect to the redis instance indicated by the args, and it deploys a GCP load balancer to handle traffic on ports 80 and 443. Traffic on :80 is needed to perform the LetsEncrypt certificate challenges, after which all SCIM traffic will be served on :443
+Within your DNS provider, create the DNS record for the domain you decided on in _Start creating your DNS record_. You should give it a few minutes to propagate. This can take anywhere from 5 minutes to an hour depending on what DNS provider you're using for the domain.
+
+You may or may not need to reboot the containers you've deployed to ensure the LetsEncrypt challege service finishes correctly, but you should see if it automatically completes on its own before doing so.
```

Comment:
Sorry why are we rebooting the containers? They should start up correctly the first time. If not, we need to fix something.